chains/nomad-ethereum/src/macros.rs

```diff
@@ -166,7 +167,8 @@ macro_rules! tx_submitter_local {
 #[macro_export]
 macro_rules! tx_submitter_gelato {
     ($base_provider:expr, $gelato_conf:ident) => {{
-        let signer = Signers::try_from_signer_conf(&$gelato_conf.sponsor).await?;
+        let kms_client = nomad_core::aws::get_kms_client().await;
+        let signer = Signers::try_from_signer_conf(&$gelato_conf.sponsor, Some(kms_client)).await?;
```

Comment:
Can we have `Signers::try_from_signer_conf` just internally call `nomad_core::aws::get_kms_client().await`? Seems repetitive to find every `try_from_signer_conf` call and add the `get_kms_client()` before it